src/nxdoc/nuxeo-server/administration/elasticsearch-setup.md

```diff
@@ -787,7 +786,7 @@ This can be done using a customized Luke tool and looking at the Lucene index le
 
 ```bash
 # view indexed tokens for dc:title.fulltext of document 3d50118c-7472-4e99-9cc9-321deb4fe053
-curl -XGET 'localhost:9200/nuxeo/doc/_search?search_type=count&pretty' -d'{
+curl -XGET 'localhost:9200/nuxeo/doc/_search?pretty' -d'{
```

Comment:
This example isn't going to work because `dc:title.fulltext` won't let you do aggregates.

Reply:
ok, I will use dc:title and warn that the field must not be text or text with fielddata